test/request_factory/test_workbook_requests.py

```diff
@@ -12,3 +15,41 @@ def test_embedded_extract_req(self):
     def test_generate_xml(self):
         workbook_item: TSC.WorkbookItem = TSC.WorkbookItem("name", "project_id")
         TSC_RF.RequestFactory.Workbook._generate_xml(workbook_item)
+
+    def test_generate_xml_invalid_connection(self):
```

Comment:
next couple tests seem to not be about sensitive data, is this just fixing code coverage misses from the past?

Reply:
yep, these cover a couple of error checks I added in request_factory while trying to write the actual test I wanted.